docker/jupyter/Dockerfile

```diff
@@ -70,19 +74,17 @@ RUN echo "deb $DEBIAN_REPO/debian stretch main"                   > /etc/apt/sou
 ENV LC_ALL en_US.UTF-8
 
 #######################
-# Java
+# GATK
 #######################
 
-ENV JAVA_VER jdk1.8.0_201
-ENV JAVA_TGZ jdk-8u201-linux-x64.tar.gz
-ENV JAVA_URL https://download.oracle.com/otn-pub/java/jdk/8u201-b09/42970487e3af4f5aa5bca3f542482c60/$JAVA_TGZ
-ENV JAVA_HOME /usr/lib/jdk/$JAVA_VER
+RUN set -e
+ENV GATK_VERSION 4.1.0.0
+ENV GATK_ZIP_PATH /tmp/gatk-${GATK_VERSION}.zip
 
-RUN wget --header "Cookie: oraclelicense=accept-securebackup-cookie" $JAVA_URL \
- && mkdir -p /usr/lib/jdk && tar -zxf $JAVA_TGZ -C /usr/lib/jdk \
- && update-alternatives --install /usr/bin/java java $JAVA_HOME/bin/java 100 \
- && update-alternatives --install /usr/bin/javac javac $JAVA_HOME/bin/javac 100 \
- && rm $JAVA_TGZ
+ #download the gatk zip
+RUN curl -L -o $GATK_ZIP_PATH https://github.com/broadinstitute/gatk/releases/download/$GATK_VERSION/gatk-$GATK_VERSION.zip \
```

Comment:
is the idea that we're installing gatk which comes with jdk instead of jdk directly?

Reply:
They're unrelated - we needed to add the GATK packages to our dockerfile, but also fix the jdk build problem that was causing the docker build to fail